addons/misc/functions/fnc_suitcaseNukeZeus.sqf

```diff
@@ -0,0 +1,211 @@
+            //         ]; 
+            //     } forEach [1,2,3];
+            // };
+            sleep 0.8;
+            { 
+                _device setObjectTextureGlobal[_x, 
+                    '#(rgb,512,512,3)text(0,1,"LCD14",0.3,"#171717","#eb4034","")'
+                ]; 
+            } forEach [1,2,3];
+            sleep 0.4;
+        };
+    };
+
+    // Activation Code
+    _device setVariable [QGVAR(suitcaseNuke_activate), _effect];
+    _device setVariable [QGVAR(suitcaseNuke_code), _customCode];
+    _device setVariable [QGVAR(suitcaseNuke_codeTarget), _target];
+    _device addMPEventHandler ["MPKilled", {
+        params ["_device"];
+        if(isServer) then {
+            switch(_device getVariable QGVAR(suitcaseNuke_activate)) do {
+                case "explosive_small": {
+                    private _explosive = createVehicle ["DemoCharge_Remote_Ammo_Scripted", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    _explosive setDamage 1;
+                    deleteVehicle _device;
+                };
+                case "explosive_large": {
+                    private _explosive = createVehicle ["Bo_GBU12_LGB", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    _explosive setDamage 1;
+                    deleteVehicle _device;
+                };
+                case "explosive_nuke": {
+                    ["zen_modules_moduleNuke", [ASLToAGL(getPosASL _unit), 200, 300, false]] call CBA_fnc_globalEvent;
+                    deleteVehicle _device;
+                };
+                case "smoke_yellow": {
+                    private _smoke = createVehicle ["SmokeShellYellow_Infinite", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    hideObjectGlobal _smoke;
+                    _device setVariable [QGVAR(suitcaseNuke_smoke), _smoke];
+                    _smoke attachTo [_device];
+                    _device addEventHandler ["Deleted", {
+                        params ["_device"];
+                        deleteVehicle (_device getVariable [QGVAR(suitcaseNuke_smoke), objNull]);
+                    }];
+                };
+                default {};
+            };
+        };
+
+        // private _customCode = _device getVariable [QGVAR(suitcaseNuke_code), ""];
+        // if(_customCode isNotEqualTo "") then {
+        //     private _codeTarget = _unit getVariable [QGVAR(suitcaseNuke_codeTarget), 0];
+
+        //     if((_codeTarget == 0 and hasInterface) || {
+        //         (_codeTarget == 1 and isServer) || {
+        //         (_codeTarget == 2)}}
+        //     ) then {
+        //         [_device] call (compile _customCode);
+        //     };
+        // };
+    }];
+
+
+    // Defusal Code
+    if(_defuser > 0) then {
+
+        // TODO: with ace, also (optionally) require DefusalKit item
+        private _ace = (isClass(configFile >> "CfgPatches" >> "ace_main"));
```

Comment:
use ``EGVAR(main,aceLoaded)`` instead. We already check and save if ACE loaded state in "main".